src/config/Config.zig

```diff
@@ -1851,6 +1851,13 @@ keybind: Keybinds = .{},
 /// must always be able to move themselves into an isolated cgroup.
 @"linux-cgroup-hard-fail": bool = false,
 
+/// Start new windows maximized. This setting applies to new windows and
+/// does not apply to tabs, splits, etc. However, this setting will apply to all
+/// new windows, not just the first one.
+///
+/// This setting is overwritten by `fullscreen` option.
```

Comment:
I think a better wording would be is that `maximized` is ignored if `fullscreen=true`.

Reply:
Changed the config description and added a note, to indicate that this feature is only for Linux, for now.